README.md

```diff
@@ -82,15 +82,15 @@ var format = d3.format(".2s");
 
 ## API Reference
 
-<a name="format" href="#format">#</a> d3.<b>format</b>(<i>specifier</i>)
+<a name="format" href="#format">#</a> d3.<b>format</b>(<i>specifier</i>) [<>](https://github.com/d3/d3-format/blob/master/src/formatDefault.js "Source")
```

Comment:
formatDefault.js contains the implementation for the (none) format type, as in `d3.format(".3")`. The correct link for d3.format is https://github.com/d3/d3-format/blob/master/src/defaultLocale.js#L4